src/components/DescriptionList.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+	<dl>
+		<div v-for="(value, name) in list" :key="name" class="listPair">
```

Comment:
`:key="name"`? Could a several names be the same?

Reply:
no, it is names of properties